CONTRIBUTING.md

```diff
@@ -42,29 +42,70 @@ Maintainers can revert your changes if they feel they are not worth maintaining
 
 As mentioned before, you are expected to follow these specifications in order to make everyone's lives easier, it will also save you and us time, with having to make the changes and us having to tell you what to change. Thank you for reading this section.
 
-* As BYOND's Dream Maker is an object oriented language, code must be object oriented when possible in order to be more flexible when adding content to it. If you are unfamiliar with this concept, it is highly recommended you look it up.
-
-* You must write BYOND code with absolute pathing, like so:
-
-```C++
-
-/obj/item/weapon/baseball_bat
-    name = "baseball bat"
-    desc = "A baseball bat."
-    var/wooden = 1
-
-/obj/item/weapon/baseball_bat/examine()
-    if(wooden)
-        desc = "A wooden baseball bat."
-    else
-        desc = "A metal baseball bat."
-    ..()
+###Object Oriented code 
+As BYOND's Dream Maker is an object oriented language, code must be object oriented when possible in order to be more flexible when adding content to it. If you are unfamiliar with this concept, it is highly recommended you look it up.
+
+###All Byond paths must contain the full path.
+(ie: absolute pathing)
+
+Byond will allow you nest almost any type keyword into a block, such as:
+
+```c++
+datum
+	datum1
+		var
+			varname1 = 1
+			varname2
+			static
+				varname3
+				varname4
+		proc
+			proc1()
+				code
+			proc2()
+				code
+		
+		datum2
+			varname1 = 0
+			proc
+				proc3()
+					code
+			proc2()
+				..()
+				code
+```
 
+The use of this is not allowed in this project as it makes finding definitions via full text searching next to impossible. The only exception is the variables of an object may be nested to the object, but must not nest further.
+
+The previous code made complaint:
```

Comment:
compliant: http://www.merriam-webster.com/dictionary/compliant

As in, it compiles with rules, to comply